Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:18.12.1
+ENV NODE_ENV=production
+
+WORKDIR /app
+COPY ["package.json", "yarn.lock", "./"]
+RUN yarn install --frozen-lockfile
+COPY . .
+RUN mv .env.example .env
```

Comment:
It would be best if you could describe exactly each step required to run the node.

Starting from:
1. clone the repo
2. something with env
3. anything else
4. docker-compose up -d

Reply:
Also - please in readme a requirement to generate an Arweave wallet - https://docs.bundlr.network/docs/participate/Types%20of%20Nodes/Validator/validator-join#arweave-wallet

To sum-up - should I simply run
1. `cp example.env .env`
2. `docker-compose up`
?

Reply:
With the above combination the images are built properly and everything seems to start:
<img width="1423" alt="image" src="https://user-images.githubusercontent.com/84311757/207640586-7178287c-0e66-428c-80a7-a48fad8f65c1.png">


but docker logs for the `warp-cache-node-dre`
show:
<img width="1308" alt="image" src="https://user-images.githubusercontent.com/84311757/207640960-090aabe7-64b9-4016-ba7c-c498e11780a7.png">

- as if the Redis didn't start properly?

Reply:
My bad, I set up BULLMQ_HOST=127.0.0.1 in the .env.example. In the docker-compose, it's set up like this:       BULLMQ_HOST: "${BULLMQ_HOST:-bullmq}". It meast, if variable is empty, use default value. 
So, the variable should be empty in the .env. :)